src/pages/signin/SignInPageLayout/BackgroundImage/index.js

```diff
@@ -1,32 +1,54 @@
 import PropTypes from 'prop-types';
 import React from 'react';
+import * as Animatable from 'react-native-animatable';
 import DesktopBackgroundImage from '@assets/images/home-background--desktop.svg';
 import MobileBackgroundImage from '@assets/images/home-background--mobile.svg';
 import useThemeStyles from '@hooks/useThemeStyles';
 import defaultPropTypes from './propTypes';
 
 const defaultProps = {
     isSmallScreen: false,
+    transitionDuration: 0,
 };
 
 const propTypes = {
     /** Is the window width narrow, like on a mobile device */
     isSmallScreen: PropTypes.bool,
 
+    /** Tranistion duration in milisecond */
+    transitionDuration: PropTypes.number,
```

Comment:
We can move the `transitionDuration` prop to https://github.com/Expensify/App/blob/ba62bdcdf1b9fd265ebecd393ae8c610ab16eb79/src/pages/signin/SignInPageLayout/BackgroundImage/propTypes.js file
`defaultPropTypes` is a confusing name, could you rename it to `backgroundImageProps`?

Reply:
@eVoloshchak I've got this lint error when trying to move it to `propTypes`.

```
/BackgroundImage/index.ios.js
  12:5  error  defaultProp "transitionDuration" has no corresponding propTypes declaration  react/default-props-match-prop-types
```

Any suggestions? or do we need to add `// eslint-disable-next-line react/default-props-match-prop-types` ?

Reply:
We can get rid of `transitionDuration` default prop I think. It isn't necessary here
That would also resolve the lint error

Reply:
Sure